condor/index.md

```diff
@@ -1,56 +1,67 @@
+| [Multi-VM Support](#multi-virtual-machine-architecture)           |                |
+| [CSPR Burning  ](#cspr-burn-function)                             | [CEP-92]()     |
+| [Native Events](#native-events)                                   | [CEP-88]()     |
+| [FFI Enhancements](#ffi-enhancements)                             |                |
+| [Expanded API Options](#expanded-api-integration-options)         |                |
+| [New Transaction Model](#new-transaction-model)                   |                |
+| [Configurable Delegation Limits](#configurable-delegation-limits) | [CEP-90]()     |
+| [Reserved Slots and Custom Fees](#reserved-slots-and-custom-fees) |                |
+| [Factory Contract Pattern](#factory-contract-pattern)             | [CEP-86]()     |
+| [Contract Access to Auction](#contract-access-to-auction)         |                |
 
-Casper v2.0 introduces a number of improvements aimed at removing some limitations of the Casper platform and dealing with technical debt. Some of the limitations in Casper 1.X were due to architectural decisions that impeded efforts to introduce new features without breaking existing systems. 
 
-#### Casper Sidecar
-In Casper 2.0, the existing RPC service has been moved outside of the node software itself. In Casper 1.X, the RPC service existed as an HTTP server which was built into the node software itself. This meant that the RPC server was tightly coupled to the node software, could not be updated without updating the node software, and executed in the same process space as the node. The result of this was that any extension of the RPC surface necessitated a full network upgrade to deliver. 
 
-In Casper 2.0, the JSON RPC has been moved to a separate process, known as the [Sidecar](https://docs.casper.network/operators/setup/casper-sidecar/). This step brings a few improvements: 
- - It allows for better process isolation and makes it easier to run and debug node processes.
- - The Sidecar provides a way to surface Contract Level Events, which opens up some interesting possibilities for node interactions.
- - The separation of the Sidecar into a new codebase means that enhancements to the RPC API can now be accomplished without changing the node binary, necessitating a network upgrade. 
+### Zug Consensus Protocol
+Casper 2.0 introduces a new consensus model known as Zug ([Whitepaper](https://arxiv.org/abs/2205.06314)). The Highway protocol is effective and secure, but resource-heavy. Zug is simpler and leaner than the Highway protocol upon which Casper was originally conceived, and as such allows for improvements in network efficiency and cohesion. This in turn facilitates eventual extension of the validator list, and finer-grained control over block times.
 
-#### Expanded API Integration Options
-Casper 2.0 introduces some extensions to the ways in which you can interact with the Casper Network.
+#### Zug in brief
```

Comment:
I think we should remove the brief section, its just a one liner and prompts more questions than it answers

Reply:
You commented on the removal (movement to a different section), but yes, I agree. Removing